### ISSUE TITLE
Why are these 2 values so much different?

### ISSUE THREAD

**Issue by inson1** [GabrielDelepine/chrome-tabs-memory-consumption#2]
Same page. Same time.
And yes I used the command (--enable-precise-memory-info).
![image](https://github.com/GabrielDelepine/chrome-tabs-memory-consumption/assets/75314629/122c900b-5bff-46a4-a9ca-49cfa412ecac)
![image](https://github.com/GabrielDelepine/chrome-tabs-memory-consumption/assets/75314629/56d0bcb2-01a8-4af8-88b3-42fdfee046c0)

**Comment by inson1** [comment]
oh, I´m idiot.

**Comment by inson1** [comment]
Its only for javascript and heap memory?

**Comment by GabrielDelepine** [comment]
Hi,

Chromium's team has done a lot of work lately on memory optimization. I think we can consider chrome-tabs-memory-consumption deprecated.

**Comment by GabrielDelepine** [comment]
> Its only for javascript and heap memory?

You're right. This extension only display `window.performance.memory.usedJSHeapSize`.

**Comment by inson1** [comment]
@GabrielDelepine Do you want to fix it?

**Comment by GabrielDelepine** [comment]
> @GabrielDelepine Do you want to fix it?

`performance.memory` [is now deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Performance/memory) and replaced by [measureUserAgentSpecificMemory](https://developer.mozilla.org/en-US/docs/Web/API/Performance/measureUserAgentSpecificMemory).

Unfortunately, this new function requires specific headers to work. Manipulating these headers within a Chrome extension is unwanted and/or forbidden. Even if it wouldn't be, that would create a security leak.

I will think about it and decommission chrome-tabs-memory-consumption if I don't find a solution. Thank you for opening this issue
